tools/castle-editor/code/framedesign.pas

```diff
@@ -725,7 +721,9 @@ function TDesignFrame.TDesignerLayer.Motion(const Event: TInputMotion): Boolean;
         end;
     end;
 
-    Frame.ModifiedOutsideObjectInspector;
+    //this will schedule recording Undo OnRelease
+    //and will prevent from recording Undo by applying changes below
+    Frame.ModifiedOutsideObjectInspector('', true);
```

Comment:
I find above 2 lines somewhat unclear. Also remember we usually make space between `//` and text in CGE comments :) I propose to change it to this:

```
{ By default ModifiedOutsideObjectInspector would record this as undo change.
  But we pass UndoOnRelease = true to defer it to the Release call. }
```

Reply:
Done, I've rewritten it in a (very) different way, please, check it out